OnPod/Node/include/sensors/lp_transducer.cpp

```diff
@@ -0,0 +1,14 @@
+#include "lp_transducer.h"
+
+U5244::U5244 (ADS7953 *adc, uint8_t uAdcChannel) :
+    adc(adc), uAdcChannel(uAdcChannel)
+    {}
+
+void U5244::init() {
+    adc->enableChannel(uAdcChannel);
+}
+
+float U5244::read() {
+    uint16_t uAdcConversion = adc->getuAdcData()[uAdcChannel];
+    return (uAdcConversion/4096*5-1)*50;
```

Comment:
if uAdcConecrsion is 0 you'll have -50 in the uint16_t field, not sure if that will get wonky or the float return will read it fine.

Reply:
It's good with the float return. There's never a negative number assigned to the uint